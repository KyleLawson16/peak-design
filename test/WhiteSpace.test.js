import React from 'react'
import { WhiteSpace } from '../build/rrpm.module'
import renderer from 'react-test-renderer'

test('WhiteSpace', () => {
	const component = renderer.create(<WhiteSpace height={20} />)

	let tree = component.toJSON()
	expect(tree).toMatchSnapshot()
})
