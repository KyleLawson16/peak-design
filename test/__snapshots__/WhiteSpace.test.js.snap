// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhiteSpace 1`] = `
<div
  style={
    Object {
      "backgroundColor": undefined,
      "height": 20,
      "width": "100%",
    }
  }
/>
`;
